html/client/config/default.json

```diff
@@ -0,0 +1,8 @@
+{
+	"Content":{
+		"contentRoute":"/content"
```

Comment:
Going to "http://biblebox.local" (I have a DNS entry for my test machine) gave me _Oh snap! Error 404 occurred, and it means “Not Found”_. Once I added a trailing slash i.e. `"contentRoute":"/content/"` everything was good.